### ISSUE TITLE
Setting suggest.autoTrigger to "none" is ignored 

### ISSUE THREAD

**Issue by chronon** [neoclide/coc.nvim#2725]
## versions

vim version: NVIM v0.4.4
node version: v12.18.3
coc.nvim version: 0.0.79-e444c57756
coc.nvim directory: /Users/chronon/.config/nvim/plugged/coc.nvim
term: xterm-kitty
platform: darwin

## Log of coc.nvim

2020-12-17T08:21:59.031 INFO (pid:61104) [services] - registered service "highlight"
2020-12-17T08:21:59.048 INFO (pid:61104) [services] - registered service "json"
2020-12-17T08:21:59.048 INFO (pid:61104) [services] - Json language server state change: stopped => starting
2020-12-17T08:21:59.086 INFO (pid:61104) [language-client-index] - highlight started with 61109
2020-12-17T08:21:59.091 INFO (pid:61104) [language-client-index] - json started with 61110
2020-12-17T08:21:59.119 INFO (pid:61104) [plugin] - coc.nvim 0.0.79-e444c57756 initialized with node: v12.18.3 after 142ms
2020-12-17T08:21:59.166 INFO (pid:61104) [services] - Json language server state change: starting => running
2020-12-17T08:21:59.168 INFO (pid:61104) [services] - service json started

## Describe the bug

In my `coc-settings.json` file I have `"suggest.autoTrigger": "none"`, but it appears to be ignored with `autoTrigger` defaulting to what looks like `all`. This coc change happened in the last day or two as I have not made any `coc-settings.json` or `init.vim` changes recently.

## Reproduce the bug

Set `"suggest.autoTrigger": "none"` in `coc-settings.json` and start typing.

## Screenshots (optional)

![Screen Shot 2020-12-17 at 8 28 59 AM](https://user-images.githubusercontent.com/57735/102493987-fc9afe00-4041-11eb-92a8-df4dce42f78f.jpg)

**Comment by chemzqm** [comment]
Try minimal vimrc.

**Comment by chemzqm** [comment]
You will have log of completion when it's triggered by coc.nvim, but they don't exists in your log.

**Comment by chronon** [comment]
I still have the issue with a minimal vimrc:

```
call plug#begin()
Plug 'neoclide/coc.nvim', {'commit': '0219b966b3355a1ff23f7927f50b5ec9a22e3dec'}
call plug#end()

set nocompatible
filetype plugin indent on
syntax on
set hidden
```

Reverting to dce1b5cb87cbafae20a24fe3b17fe9648e84dd4e fixes the issue for me, apparently making the `"suggest.autoTrigger": "none"` do what it's supposed to do. Checking out the next commit 0219b966b3355a1ff23f7927f50b5ec9a22e3dec is where it breaks for me.

**Comment by chemzqm** [comment]
Try completion in your home directory, you may have local settings file.

**Comment by chronon** [comment]
I see in #2728 someone else ran into the same issue, and your fix in adf219e77bfe84da3f3d570afcb8c4889d4e3daa fixed this for me as well. Thank you!
